### PR TITLE
Add a constructor than picks int[] arrays.

### PR DESCRIPTION
**Pull request by RaphMar2021**
Added a constructor than picks int[] arrays.

### CODE REVIEW

File: source/Cosmos.System2/Graphics/Bitmap.cs [hunk 1]
```diff
@@ -55,6 +55,15 @@ public Bitmap(uint Width, uint Height, byte[] pixelData, ColorDepth colorDepth)
             }
         }
         
+        public Bitmap(uint Width, uint Height, int[] pixelData, ColorDepth colorDepth) : base(Width, Height, colorDepth)
```
Comment (quajak) [inline source/Cosmos.System2/Graphics/Bitmap.cs:58]: Please add documentation to the function call. I also dont think this function call needs ColorDepth and can always use ColorDepth32

**Review by quajak — CHANGES_REQUESTED** [review]
Thanks for the contribution!

**Comment by TheFocusMan** [comment]
This PR is old

**Comment by RaphMar2021** [comment]
i don't care

**Comment by RaphMar2021** [comment]
@TheFocusMan

**Comment by TheFocusMan** [comment]
> i don't care

Why is not approved yet?

**Comment by RaphMar2021** [comment]
idk
ping a coredev on cosmos discord because i left it

**Comment by TheFocusMan** [comment]
Bro u should to request review and i am not even in the discord server

**Comment by RaphMar2021** [comment]
bye

**Pull request closed without merging**